pkg/konfluxgen/fbc-builder.yaml

```diff
@@ -115,12 +115,6 @@ spec:
     name: CHAINS-GIT_COMMIT
     value: $(tasks.clone-repository.results.commit)
   tasks:
-  - name: fbc-related-image-check
-    when:
-    - input: $(params.skip-fbc-related-image-check)
-      operator: in
-      values:
-      - "false"
   - name: build-container
```

Comment:
Does `build-container` really work? It doesn' reference any task. I suppose it will fail.

Reply:
I guess we can remove the FBC patches: https://github.com/openshift-knative/hack/pull/465#discussion_r1873090669